### PR TITLE
Improve resolve on windows

### PR DESCRIPTION
**Pull request by ahaoboy**
### Description of changes

The following test fails on Windows due to the path separator

```
  _require(`${CWD}/fixtures/test_modules/test-uuid.js`);
```

### Checklist

- [x] Created unit tests in `tests/unit` and/or in Rust for my feature if needed
- [x] Ran `make fix` to format JS and apply Clippy auto fixes
- [x] Made sure my code didn't add any additional warnings: `make check`
- [ ] Added relevant type info in `types/` directory
- [ ] Updated documentation if needed ([API.md](API.md)/[README.md](README.md)/Other)

### CODE REVIEW

File: llrt_core/src/module_loader/resolver.rs [hunk 1]
```diff
@@ -73,7 +73,13 @@ impl Resolver for CustomResolver {
...
         trace!("Try resolve '{}' from '{}'", name, base);
 
-        require_resolve(ctx, name, base, true).map(|name| name.into_owned())
+        require_resolve(ctx, name, base, true).map(|name| {
+            if cfg!(not(windows)) {
+                name.into_owned()
+            } else {
+                name.replace("\\", "/")
+            }
+        })
```
Comment (richarddavison) [inline llrt_core/src/module_loader/resolver.rs:82]: We have a "to_abs_path" used by `require_resolve` in various places. Use that method inside of require resolve where this case isn't covered!

**Review by richarddavison — CHANGES_REQUESTED** [review]

**Comment by ahaoboy** [comment]
> The results returned by `require_resolve()` (build-in module name, bytecode cache name, absolute path including search results from package.json, presence or absence of prefix used on the loader side, etc.) vary, so I think it's difficult to judge here.
> 
> I'll take a look at this if you don't mind. :)

The logic is indeed a bit complicated, and I'm not sure whether it should be handled here. If there is a better way, you can directly submit a new PR

**Comment by nabetti1720** [comment]
@ahaoboy , This issue was addressed in #690. We have confirmed that the Require tests pass at least in windows-build.

image

**Comment by nabetti1720** [comment]
@ahaoboy, #690 has been merged into mainline, so this PR is no longer needed. :)

**Pull request closed without merging**